README.md

```diff
@@ -1,4 +1,4 @@
-# Deploy to iTunes Connect with Deliver
```

Comment:
I have left similar comment on your other PR so I will not go into details and just mark it here as well. 😄

```
Deploy to App Store Connect (formerly iTunes Connect) with Deliver
```

Here and in the `step.yml`.